src/Core/Addon/Module/ModuleRepository.php

```diff
@@ -339,6 +352,15 @@ public function getModule($name)
     }
 
     /**
+     * Function which returns the modules directory. Used for mock in tests/ folder.
+     * @return string The modules directory
+     */
+    protected function getModulesDir()
```

Comment:
wow please :'(

you can "mock" constants using phpunit configuration (https://phpunit.de/manual/current/en/phpunit-book.html#appendixes.configuration.php-ini-constants-variables) or in beforeClass/setUp() functions.